### PR TITLE
Fixed and extended login page tests

### PR DESCRIPTION
**Pull request by jrichter1**
Plus some IDs added to used elements.

### CODE REVIEW

File: browser/pages/account/account.html [hunk 1]
```diff
@@ -24,21 +24,21 @@
...
-            <strong>Terms and Conditions for CDK have not been signed.</strong>
+          <div id="tcError" ng-show="accountForm.$submitted && acctCtrl.tandcNotSigned" class="help-block">
+            <span id="tcIcon" class="pficon pficon-error-circle-o"></span>
+            <strong id="tcMessage">Terms and Conditions for CDK have not been signed.</strong>
           </div>
         </div>
         <div class="form-group pull-right form-buttons">
-          <button type="button" class="btn-plain btn-spacer btn-med-size btn btn-lg" ng-click="acctCtrl.createAccount()">Register</button>
-          <button type="submit" class="btn-med-size btn btn-primary btn-lg" ng-disabled="accountForm.$invalid">Log in</button>
+          <button id="register" type="button" class="btn-plain btn-spacer btn-med-size btn btn-lg" ng-click="acctCtrl.createAccount()">Register</button>
```
Comment (ppitonak) [inline browser/pages/account/account.html]: What about using the same naming conventions, i.e. `registerButton`

File: test/ui/login-test.js [hunk 2]
```diff
@@ -1,78 +1,122 @@
...
+  let usernameField, passwordField, loginButton;
 
-describe('basic ui test', function() {
+  beforeAll(function() {
+    webdriver.wait(protractor.until.elementLocated(By.id('loginButton')), 10000)
+    .then(function(elm) {
+      loginButton = elm;
+    });
+    usernameField = element(By.id('username'));
+    passwordField = element(By.id('password'));
```
Comment (ppitonak) [inline test/ui/login-test.js]: Why are these two outside of `then` function?

File: test/ui/login-test.js [hunk 3]
```diff
@@ -1,78 +1,122 @@
...
+    });
+
+    afterAll(function() {
+      usernameField.clear();
+      passwordField.clear();
+    });
+
+    it('Username field should be focused', function() {
+      let activeElmId = webdriver.switchTo().activeElement().getAttribute('id');
+      expect(usernameField.getAttribute('id')).toEqual(activeElmId);
```
Comment (ppitonak) [inline test/ui/login-test.js]: expected value and actual value are swapped

File: test/ui/login-test.js [hunk 4]
```diff
@@ -1,78 +1,122 @@
...
+    });
+
+    it('Leaving empty password field should display an error', function() {
+      usernameField.sendKeys('');
+      expect(usernameStatus.isDisplayed()).toBe(true);
+    });
+
+    it('Entering a username should hide the appropriate error', function() {
+      //ensure the error messages are present by switching between inputs
+      usernameField.sendKeys('');
```
Comment (ppitonak) [inline test/ui/login-test.js]: is this necessary?

File: test/ui/login-test.js [hunk 5]
```diff
@@ -1,78 +1,122 @@
...
+      passwordField.sendKeys('');
+
+      usernameField.sendKeys('user');
+      expect(usernameStatus.isDisplayed()).toBe(false);
+      usernameField.clear();
+    });
+
+    it('Entering a username should hide the appropriate error', function() {
+      //ensure the error messages are present by switching between inputs
+      passwordField.sendKeys('');
```
Comment (ppitonak) [inline test/ui/login-test.js]: is this necessary?

**Comment by jrichter1** [comment]
Updated and rebased.

**Pull request merged**